test/exclusions/corefx/linux.docker.rsp

```diff
@@ -0,0 +1,61 @@
+# Ignore the tests that are to be circumvented for CI
+-notrait category=IgnoreForCI
+
+# Ignore any other failing tests for SB coreFX tests
+-notrait category=failing
+
+# System.Diagnostics.Tests.ProcessTests
```

Comment:
Why all these static lists?  /cc @ViktorHofer

Reply:
Oh good catch! I can try it without mentioning the failed tests separately. I think `category=failing` should take care of it.